datastore-v1-proto-client/src/test/java/com/google/datastore/v1/client/ChecksumEnforcingInputStreamTest.java

```diff
@@ -66,9 +63,7 @@ public void read_withInvalidChecksum() {
     // Since checksum should be correct, do not expect IOException
     try (ChecksumEnforcingInputStream instance =
         new ChecksumEnforcingInputStream(
-            new ByteArrayInputStream("hello there".getBytes(UTF_8)),
-            "this checksum is invalid",
-            digest)) {
```

Comment:
it seems odd to me that there is no digest. Is this different than the checksum from crc32c? Is this just internal now somehow?

Reply:
yes sir - CRC32C doesn't have/need digest..

Chris, Kristen - you know you guys are *most* welcome to recommend MD5, instead of CRC32C, for clients.
MD5 is way stronger and is readily available in all Java versions (whereas CRC32C is not in core java libs until JDK 11)